### DISCUSSION TITLE
提问题的注意事项

### DISCUSSION THREAD

**Discussion by lxhtt** [MCSLTeam/MCSL2#57] [常见问题]
首先，请注意以下图片：
![Apache名言](https://github.com/LxHTT/MCSL2/assets/98154001/56395612-d394-4f7e-a6ba-483b00a761d0)
![截图](https://github.com/LxHTT/MCSL2/assets/98154001/849dc740-76f2-456f-afca-715756e4aae7)
___
咳咳咳。  
请务必以友善的态度发问，才比较容易得到他人的解答。  
解答时，如果有奇奇怪怪的东西出现，也可以at LxHTT。
请务必在“提问区域”发文哦。
然后也没啥好说的了。。。

**Comment by ltzXiaoYanMo** [comment]
没啥好说的那就再加一个（？）
![2CA01736A88A3AEF9621E1A14DCA2E29](https://github.com/MCSLTeam/MCSL2/assets/115706851/eeaf9b38-350b-4944-81c0-a8a6d21d53e0)
